recipes/pynvml/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2011-2019, NVIDIA Corporation.
```

Comment:
Submitted PR ( https://github.com/gpuopenanalytics/pynvml/pull/25 ) upstream to make sure this is included in `pynvml` itself.

Reply:
Also added a reminder about updating the feedstock accordingly with issue ( https://github.com/conda-forge/pynvml-feedstock/issues/5 ).